src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -0,0 +1,56 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import _ from 'lodash/fp';
+import { createCriteriaFromType, renderCriteriaView } from 'src/pages/library/datasetBuilder/CohortEditor';
+
+describe('CohortEditor', () => {
+  it('renders unknown criteria', () => {
+    const criteria = { bogus: 'criteria' };
+    // @ts-ignore
```

Comment:
Do you know why you need a ts-ignore here? Is it just to make sure you can pass in a criteria type that doesn't match any of the known criteria as a prop?

Reply:
> Do you know why you need a ts-ignore here? Is it just to make sure you can pass in a criteria type that doesn't match any of the known criteria as a prop?

Reply:
in rare cases where this sledgehammer is needed, we should at least specify the specific lint rule being circumvented, which is easier to do with the "es-lint ingnor next line" comment pattern, which lets you specify the exact es-lint rule code to ignore.

But here there is a better option... I would do this to make it explicit you want your test to be "misbehaving":

// purposefully bogus input value
const bogusCriteria: TheCriteriaType = { bogus: 'criteria' } as unknown as TheCriteriaType;